views/purchase_order_enhancement.xml

```diff
@@ -0,0 +1,88 @@
+                    type="object" 
+                    name="action_unarchive_purchase_orders"
+                    class="oe_stat_button"
+                    icon="fa-archive" 
+                    groups="purchase.group_purchase_manager" />
+            </xpath>
+            <xpath expr="//page[@name='purchase_delivery_invoice']" position="after">
+                <page string="Cron Information" name="cron_info">
+                    <group>
+                        <field name="lifespan"></field>
+                    </group>
+                </page>
+            </xpath>
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="usphone" widget="usphone_format"></field>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="action_archive_purchase_orders" model="ir.actions.server">
+        <field name="name">Archive Purchase Orders</field>
+        <field name="model_id" ref="model_purchase_order"></field>
+        <field name="binding_model_id" ref="model_purchase_order"></field>
+        <field name="binding_view_types">list</field>
+        <field name="state">code</field>
+        <field name="code">
+            if records:
+                records.action_archive_purchase_orders()
+        </field>
+    </record>
+
+    <record id="action_unarchive_purchase_orders" model="ir.actions.server">
+        <field name="name">Unarchive Purchase Orders</field>
+        <field name="model_id" ref="model_purchase_order"></field>
+        <field name="binding_model_id" ref="model_purchase_order"></field>
+        <field name="binding_view_types">list</field>
+        <field name="state">code</field>
+        <field name="code">
+            if records:
+                records.action_unarchive_purchase_orders()
+        </field>
+    </record>
+
+    <record id="purchase_order_enhancement_search_view" model="ir.ui.view">
+        <field name="name">purchase.order.enhancement.search.view</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_view_search"></field>
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='starred']" position="after">
+                <filter name="is_archived" string="Is Archived" domain="[('active', '=', False)]"></filter>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_order_enhancement_tree_view" model="ir.ui.view">
+        <field name="name">purchase.order.enhancement.tree.view</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_view_tree"></field>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="inside">
+                <field name="active" invisible="1"></field>
+            </xpath>
+            <xpath expr="//tree" position="attributes">
+                <attribute name="decoration-muted">active==False</attribute>
```

Comment:
This tree view already defined the decoration-muted so please make sure you keep its origin